model_analyzer/config/run/run_config_generator.py

```diff
@@ -107,51 +125,178 @@ def _generate_model_config_combinations(self, value):
         # always return a list.
         return [value]
 
-    def _generate_run_configs(self):
+    def execute_run_configs(self):
```

Comment:
Do you think we can extract out a class called `RunConfigExecutor` for this? This config generationcan happen in the current class but the execution may be able to be moved

Reply:
I'll refactor this later. It takes a long time to do this.